### PR TITLE
Fix grammar mistake

### PR DESCRIPTION
**Pull request by ocrampete16**
Just something I stumbled upon while reading through the docs.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

File: http_cache/esi.rst [hunk 1]
```diff
@@ -95,7 +95,7 @@ First, to use ESI, be sure to enable it in your application configuration:
 
 Now, suppose you have a page that is relatively static, except for a news
 ticker at the bottom of the content. With ESI, you can cache the news ticker
-independent of the rest of the page::
+independently of the rest of the page::
```
Comment (zanbaldwin) [inline http_cache/esi.rst]:
```suggestion
independently from the rest of the page::
```

File: http_cache/esi.rst [hunk 2]
```diff
@@ -158,7 +158,7 @@ used ``render()``.
     specification that is supported out of the box by the Symfony reverse
     proxy.
 
-The embedded action can now specify its own caching rules, entirely independent
+The embedded action can now specify its own caching rules entirely independently
 of the master page::
```
Comment (zanbaldwin) [inline http_cache/esi.rst]:
```suggestion
from the master page::
```

**Review by zanbaldwin — COMMENTED** [review]
I've always been taught to use *from* instead of *of* but I'm not sure how much that depends on region/dialect of English :confused:

**Comment by xabbuh** [comment]
@zanbaldwin I think according to https://www.oxfordlearnersdictionaries.com/definition/english/independent_1 "independent of" is correct here.

**Comment by javiereguiluz** [comment]
This proposed change is really tricky. As someone posted on Symfony's Slack, books use "independently from" more than "independently of" (see https://books.google.com/ngrams/graph?content=independently+of%2Cindependently+from&year_start=1800&year_end=2008&corpus=0&smoothing=3&share=&direct_url=t1%3B%2Cindependently%20of%3B%2Cc0%3B.t1%3B%2Cindependently%20from%3B%2Cc0).

But also, we have different opinions about this from native English speakers. Zan here leans towards "independently from" and Neal on Slack defends the use of "independently of". Both of them are native in English! So I don't know what to do :|

**Comment by zanbaldwin** [comment]
Both [Google Trends](https://trends.google.com/trends/explore?geo=GB&q=independently%20of,independently%20from) and [Google Books](https://books.google.com/ngrams/graph?content=independently+of%2Cindependently+from&year_start=1800&year_end=2008&corpus=0&smoothing=3&share=&direct_url=t1%3B%2Cindependently%20of%3B%2Cc0%3B.t1%3B%2Cindependently%20from%3B%2Cc0) seem to favour "*independently of*".

My suggestion of "*independently from*" is probably just a regional thing, so I would suggest sticking to what is most widely used ("*independently of*").

**Comment by ocrampete16** [comment]
I suppose it's a regional thing. Personally, both feel right to me (native English speaker) but I would probably lean a more towards _independent of_ as well.

**Comment by javiereguiluz** [comment]
This is merged now! Thanks @ocrampete16 and congrats on your first Symfony Docs contribution! Thanks also to reviewers and to our English native friends who helped us decide what to do here.

**Pull request closed without merging**